api/src/main/java/com/velocitypowered/api/proxy/config/ProxyConfig.java

```diff
@@ -62,6 +62,15 @@
    */
   boolean isOnlineMode();
 
+  /**
+   * If client's ISP/AS sent from this proxy is different from the one from Mojang's"
+   * authentication server, the player is kicked. Effectively disallows players from
```

Comment:
"Effectively" is too strong. I'd do "This disallows some VPNs and proxy connections but is a weak form of protection."